### ISSUE TITLE
Expose APKs directly on the website

### ISSUE THREAD

**Issue by bakulf** [mozilla-mobile/mozilla-vpn-client#556]
@rlr this can be an alternative for android users. They can choose between using google play and the APK directly.
See for instance this: https://www.whatsapp.com/download/ and in particular: https://www.whatsapp.com/android/

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-103)

**Comment by rlr** [comment]
This has been discussed previously and it was decided not to do it. We'd need to weight the pros and cons to see if it makes sense to build out this flow for users. It also adds eng/releng/ops work for each release. We definitely need product and support input here. And UX/design input if we decide to move forward with it.

**Comment by twitwi** [comment]
I think this is an important point. I'm obviously biased as I'm using an android phone with all google services unused/disabled (so no play store). I hope I'm not the only one though, especially in the pool of people interested by Mozilla VPN.

I had to build it for my manjaro linux and this went relatively fine, with the proper system packages.
I tried to build the apk but failed to do so for now (I'm pretty sure it will work but it will take some extra exploration and downloads).

Signal is also a good example of apk app that "self-updates" https://signal.org/android/apk/
